src/3rd_party/apache-log4cxx/helpers.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+#
```

Comment:
@mkorniichuk please add to comment main functions and required variables, and some usage examples

Reply:
@LuxoftAKutsan fixed within bf0ee058043a3c5988f9ca2f28ecf5c31183842f

Reply:
@AByzhynar cmake will use this script, not a developer, so a simple comment is enough.